### PR TITLE
add more attributes to localdisk

### PR DESCRIPTION
**Pull request by AmazingPangWei**
#### What this PR does / why we need it:

issue: #984

#### Does this PR introduce a user-facing change?

```release-note
add more attributes to localdisk as shown below:

- major
- minor
- devLinks
```

### CODE REVIEW

File: pkg/apis/hwameistor/v1alpha1/localdisk_types.go [hunk 1]
```diff
@@ -65,6 +65,12 @@ type DiskAttributes struct {
 
 	// Protocol is for data transport, such as ATA, SCSI, NVMe
 	Protocol string `json:"protocol,omitempty"`
+
+	// Major represents drive used by the device
+	Major string `json:"major,omitempty"`
+
+	// Minor is used to distinguish different devices
+	Minor string `json:"minor,omitempty"`
```
Comment (SSmallMonster) [inline pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: The `Major:Minor` is good for me.

BTW, the value of Major:Minor is determined by the driver, whether it is better to define it at the upper level?

Comment (AmazingPangWei) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: Actually, I'm confused about spec.diskAttributes and don't know when should I put attributes into it.

Comment (SSmallMonster) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: `DiskAttributes` represent certain hardware/static attributes of the disk.

Some properties of the disk itself are suitable here, and these properties will not change as the system changes or administrator actions (for example, inserting or removing disks will cause DEVPATH changes).

Comment (AmazingPangWei) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: Got it. I'll fix this part.

Comment (AmazingPangWei) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: Is there any other mistakes should I fix?

Comment (SSmallMonster) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go:73]: No, thank you. Once it ready, the e2e test passes and it can be merged.

Comment (AmazingPangWei) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go]: Review again, plz.

Comment (SSmallMonster) [reply pkg/apis/hwameistor/v1alpha1/localdisk_types.go]: Thanks, this is good to me. The e2e test is running now.

**Comment by SSmallMonster** [comment]
lgtm

**Pull request merged**